Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -1262,6 +1264,10 @@ define([
             uniformMapProperties.minMaxHeight.y = encoding.maximumHeight;
             Matrix4.clone(encoding.matrix, uniformMapProperties.scaleAndBias);
 
+            if (tileProvider.uniformMap) {
+                 uniformMap = combine(uniformMap, tileProvider.uniformMap);
+            }
```

Comment:
It would be best to combine uniforms elsewhere since this will allocate a uniform map object every frame.

I wonder if this could be moved to the area near `if (tileProvider._drawCommands.length <= tileProvider._usedDrawCommands) {`. Can it detect when the material is dirty and combine up there?